src/System.Threading/tests/ReaderWriterLockSlimTests.cs

```diff
@@ -383,6 +383,7 @@ public static void WriterToUpgradeableReaderChain()
 
         [Fact]
         [OuterLoop]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17443")] // Hangs in desktop
```

Comment:
Sounds good I will update the comment to point to the original issue and PR and will close the issue that I opened.